imagedephi/redact.py

```diff
@@ -90,6 +161,96 @@ def execute_plan(self) -> None:
                 rule.apply(ifd)
 
 
+class SvsMetadataRedactionPlan(TiffMetadataRedactionPlan):
+    """
+    Represents a plan of action for redacting files in Aperio (.svs) format.
+
+    Redaction for this type of file is similar to redaction for .tiff files, as the
+    formats are similar. However, Aperio images store additional information in its
+    ImageDescription tags. As a result, this tag is treated specially here.
+    """
+
+    file_format = FileFormat.SVS
+    description_redaction_steps: dict[str, MetadataSvsRule]
+    no_match_description_keys: set[str]
+
+    def __init__(
+        self,
+        tiff_info: TiffInfo,
+        base_rules_tiff: list[MetadataTiffRule],
+        base_rules_svs: list[MetadataSvsRule],
+        override_rules_tiff: list[MetadataTiffRule],
+        override_rules_svs: list[MetadataSvsRule],
+    ) -> None:
+        super().__init__(tiff_info, base_rules_tiff, override_rules_tiff)
+
+        image_description_tag = tifftools.constants.Tag["ImageDescription"]
```

Comment:
We have some mislabeled or malformed tiff files that do NOT have ImageDescription tags.  These fail:
```
  File "/home/manthey/misc/ImageDePHI/imagedephi/redact.py", line 74, in build
    return SvsMetadataRedactionPlan(
  File "/home/manthey/misc/ImageDePHI/imagedephi/redact.py", line 188, in __init__
    del self.redaction_steps[image_description_tag.value]
KeyError: 270
```
We should more gracefully refuse to process them (or handle them as plain tiff files).